### ISSUE TITLE
build for mobile apps

### ISSUE THREAD

**Issue by mike623** [jedwards1211/meteor-webpack-react#118]
just wanna know how to build the app for android/ios

**Comment by gdub01** [comment]
I'm wondering this too. Just found this repo https://github.com/spencercarli/meteor-todos-react-native with a connection to dpp. But it uses a completely separate code base than Meteor. (naturally)

Then there's this one: https://github.com/benoitvallon/react-native-nw-react-calculator that has react native and web in the same codebase. They swap out the renders depending on which device you build, but keep the other logic.

So what would be a best practice? I'm guessing use the same codebase and just swap the render methods during the build and then connect to dpp on the route level?

**Comment by Mokto** [comment]
You can use https://github.com/inProgress-team/react-native-meteor plugin if it helps you.

It brings reactivity to you ;)

**Comment by mike623** [comment]
just tried to modified dev.js to " meteor run ios-device and got "SyntaxError: Unexpected token '<'" on devices 
![screen shot 2015-12-05 at 3 04 11 am](https://cloud.githubusercontent.com/assets/14008281/11598451/e117f628-9afc-11e5-856a-ebe614f37a56.png)

is the problem of es6 support?

**Comment by AdamBrodzinski** [comment]
I'm going to *try* to check this out over the weekend as I have a meteor cordova project now. I'll post back here with details.

**Comment by OminStyle** [comment]
:+1:

**Comment by mike623** [comment]
any gd news @AdamBrodzinski ?

**Comment by AdamBrodzinski** [comment]
@mike623  sorry my free time slipped away :frowning:  I doubt i'll be doing this in the near future.

**Comment by mike623** [comment]
nvm take ur time lol

**Comment by jedwards1211** [comment]
I haven't used `react-native` but from what I've seen it's different enough that a completely different app skeleton would be needed to use it with Meteor.  Doesn't `react-native` have its own build toolchain?  That might make it more difficult to use with Meteor...

**Comment by AdamBrodzinski** [comment]
@jedwards1211 yeah it's a different app structure in general. You basically replace AJAX calls with DDP methods and subscriptions. I just used a REST API on a Meteor server for simplicity (albeit not realtime)

**Comment by jedwards1211** [comment]
Right, though when starting a Meteor web app from the ground up I use DDP methods instead of AJAX calls anyway :)  But that's not really the main difference I'm talking about, rather doesn't `react-native` package your files up and send them to the device itself?  I'm not sure how we would work Meteor into that process.

**Comment by Mokto** [comment]
Guys,

Please check [react-native-meteor](https://github.com/inProgress-team/react-native-meteor).

1.0.0 version is helping you to write your react-native applications exactly like meteor-webpack-react.

:)

**Comment by mike623** [comment]
@Mokto wow! Seem it is a great project. Thx bro
